invenio_communities/subcommunities/services/request.py

```diff
@@ -91,3 +177,18 @@ def subcommunity_request_type(app):
     if not app:
         return
     return app.config.get("COMMUNITIES_SUB_REQUEST_CLS", SubCommunityRequest)
+
+
+def subcommunity_invitation_request_type(app):
+    """Return the subcommunity request type.
+
+    Since it can be overridden by the application, this function should be used
+    as the entry point for the request type.
+
+    It must return a class that inherits from `RequestType`.
```

Comment:
Minor: If we add this comment, maybe it's worth to actually check the type to ensure it is correct